### PR TITLE
[DOCS-446] Add getLastActivityDate and getStatus to Device

### PR DESCRIPTION
**Pull request by vl4ds**
These methods are currently undocumented - adding docs:
getLastActivity returns the date of the last event with a source of 'Device'

getStatus returns the status of the device - i.e. ONLINE, ACTIVE, INACTIVE.. if the device doesn't have a status then INACTIVE will be returned.

### CODE REVIEW

**Comment by vl4ds** [comment]
cc @jimmyjames @rajkaramchedu  
fyi I'm having problems getting make html to work when setting up the docs so I haven't been able to verify this looks fine locally. If either of you have a few minutes on Monday would you mind helping me troubleshoot? Thanks!

File: ref-docs/device-ref.rst [hunk 1]
```diff
@@ -528,6 +541,19 @@ Not applicable for cloud or LAN-connected devices (``null`` will be returned).
...
+getStatus()
+-----------
+
+Get the current status of the Device. If no status is found then ``INACTIVE`` is returned.
+
+**Signature:**
+    ``String getStatus()``
+
+**Returns:**
+    `String`_ - the status of the Device.
```
Comment (jimmyjames) [inline ref-docs/device-ref.rst]: Can we include the possible return values if we know them (as mentioned in the description of this PR)?

**Comment by vl4ds** [comment]
Made the suggested changes. Thanks for the review.

**Comment by jimmyjames** [comment]
Awesome. This looks good to me but @jackchi should verify

**Comment by jimmyjames** [comment]
@mortent maybe you can verify since @jackchi appears to be ooo. Thanks!

**Pull request merged**